en/docs/develop-components/develop-components-with-git.md

```diff
@@ -48,6 +48,13 @@ Authorizing Choreo as a Bitbucket application provides Choreo the following perm
 |Pull Requests | Y   | Y    | Create a pull request if you have chosen to start with a Choreo sample |
 |Webhooks      | Y   | Y    | Trigger automatic deployment and configuration generation              |
 
+## Authorize GitLab Self-Managed with Choreo
```

Comment:
Shall we say " Authorize Self-Managed GitLab with Choreo"

Reply:
Addressed in https://github.com/YashodhaSamarawickrama/docs-choreo-dev/pull/1